installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,30 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, the only authentication mechanism available to users is authentication via Red Hat SSO.
+Red Hat SSO auth provider cannot be deleted or changed.
```

Comment:
Default RHSSO auth provider cannot be deleted and *some* of its settings cannot be changed. This is to prevent users from locking themselves out of their ACS instance.

However, we strongly encourage users to set up their own IdP. Pro tip: the name of their auth provider is ideally lexicographically smaller than "Red Hat ..." so their IdP comes first and is hence the default one on the ACS console login screen.

Reply:
> You can't delete the Red Hat SSO or change its settings, but is it true that you can configure a different IdP (after the initial setup)?

Correct, you can configure a different IdP once you accessed the instance via Red Hat SSO.